databases/week3/hyf_homework_databases_lesson3.sql

```diff
@@ -0,0 +1,166 @@
+insert into meal (title, description, location, `when`, max_reservations, price, created_date) 
+	values ('Musakka', 'Preheat your oven to 220 degrees celsius. Remove the stalks from your aubergines and slice lengthways - around 1 cm thick. Lay your aubergine slices onto an oven tray and lightly drizzle with olive oil. Bake in the oven for approx 15 minutes until the aubergines are softening and have taken on colour. Remove from the oven and allow to cool. Meanwhile, heat a dash of olive oil over a medium heat in a frying pan. Add your onion and peppers and saute gently until the onions start to sweat. Now add your meat and garlic and stir in. Saute until your meat browns and starts to release its juices. Now you can add your kekik and paprika and mix. Add your chopped tomatoes, stir and cook for 5 minutes until your tomatoes start to soften. Pour in around 50ml warm water and add your salça and mix. Simmer for 5 minutes, do a taste test and season to taste with salt and pepper. Add more water if you want a thinner sauce and simmer for 15-20 mins until you have a rich tomato sauce. Now return to your aubergines. Remove from your baking tray and cut each length into about 4 pieces. Now take an oven proof dish and lay your aubergines over the base. Pour the meat mixture over the top along with a cup of hot water. Place in the centre of your oven at around 200 degrees celsius. Cook for 20 minutes until youır sauce is bubbling. Remove from the oven and leave to stand to cool a little before garnishing with fresh chopped parsley. Serve on its own or with a side of rice or mashed potato.', 'Ankara', NOW(), 8, 120, '2021-04-02');
+
+insert into reservation (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) 
+	values (10, 5, '2021-06-22', 0521321421, 'Carl Max', 'carlmax@gmail.com');
+insert into reservation (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) 
+	values (3, 6, '2021-07-12', 921514216, 'Bill Gates', 'bill@hotmail.com');
+insert into reservation (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) 
+	values (1, 2, '2021-03-11', 14215161, 'Cristiano Ronaldo', 'cr7@gmail.com');
+
+insert into review (title, description, meal_id, stars, created_date) 
+	values ('Expensive', 'I do not mind spending a lot of money but 4 pieces of yaprak sarma for 40$ that was hilarious.', 4, 2, '2021-04-22');
+insert into review (title, description, meal_id, stars, created_date) 
+	values ('Bad', 'This place is a joke...', 5, 1, '2021-04-21');
+insert into review (title, description, meal_id, stars, created_date) 
+	values ('Nice', 'Meal was quite good.', 5, 3, '2021-04-22');
+
+SELECT *
+FROM meal
+WHERE price < 90;
+
+SELECT meal.title, meal.max_reservations, reservation.number_of_guests, reservation.created_date
+FROM meal
+JOIN reservation ON
```

Comment:
What about meals without reservations yet?

Reply:
I tried to fix it, did I get it right?

Reply:
Yep! Looks good to me, well done! :)